corehq/apps/app_manager/templates/app_manager/partials/releases/app_release_logs.html

```diff
@@ -0,0 +1,43 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<div class="panel panel-appmanager">
+  <div class="panel-body">
+    <table id="release-logs-table" class="table table-profiles" >
+      <thead>
+        <tr class="row form-group">
+          <th>{% trans "Version Number" %}</th>
+          <th>{% trans "Update Type" %}</th>
+          <th>{% trans "Time" %}</th>
+          <th>{% trans "User" %}</th>
+        </tr>
+      </thead>
+      <tbody data-bind="foreach: releaseLogs">
+        <tr class="row">
+          <td class="col-sm-3">
+            <span data-bind="text: version"></span>
+          </td>
+          <td class="col-sm-3">
+            <span data-bind="visible: is_released()">{% trans "Released" %}</span>
+            <span data-bind="visible: !is_released()">{% trans "In Test" %}</span>
+          </td>
+          <td class="col-sm-3">
+            <span data-bind="text: timestamp"></span>
+          </td>
+          <td class="col-sm-3">
+            <span data-bind="text: user_email"></span>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <pagination data-apply-bindings="false"
+                params="goToPage: goToPage,
+                        perPage: fetchLimit,
+                        slug: 'release-logs-table',
+                        totalItems: totalItems,
+                        onLoad: onPaginationLoad,
+                        showSpinner: showPaginationSpinner"></pagination>
+
+  </div>
+</div>
```

Comment:
nit: I think you would need to put a blank line here. I believe that is already by editor configs we have setup for use
https://github.com/dimagi/commcare-hq/blob/9cacc528b3d098d2ef0c1bedd76dcff0da3bbe2f/.editorconfig#L7